.github/workflows/go.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and test Go
+
+on:
+  push:
+    branches:
+    - "master"
+  pull_request:
+    branches:
+    - "master"
+env:
+    GO111MODULE: on
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v2
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.12.x
```

Comment:
Maybe not related to this PR, but I think we should consider using a more recent version of Go.

Reply:
Agreed. I used 1.12 as this is the documented/tested go version for the chaincode, but we should move to 1.15.